docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "2"
 
 services:
   app:
-    build: .
```

Comment:
Possible better to keep both `build:` and `image:` as per https://docs.docker.com/compose/compose-file/ 

> If you specify image as well as build, then Compose names the built image with the webapp and optional tag specified in image